src/components/atoms/index.js

```diff
@@ -1,16 +1,14 @@
-//> Components
```

Comment:
There are no atoms yet. Please remove the folder `src/components/atoms` and re-add it as soon as you create atoms.

Reply:
We preferred to create the structure with this branch, to make further contributing easier.